### DISCUSSION TITLE
Hosting multiple server sessions?

### DISCUSSION THREAD

**Discussion by ajay-collab** [bluenviron/mediamtx#256] [Questions]
Hi, is there a way to host multiple server sessions, on different ports, if yes could you please inform as to how to do it

**Comment by aler9** [comment]
Hi, of course you can host multiple instances of the server, just change the 3 ports needed by RTSP. The only requisite is that the RTP port must be even, and the RTCP port must be just after the RTP port.

instance 1
```
rtsp-simple-server --rtsp-port=8554 --rtp-port=8000 --rtcp-port=8001
```

istance 2
```
rtsp-simple-server --rtsp-port=8555 --rtp-port=8002 --rtcp-port=8003
```

[a comment by an automated account is not shown]
